src/data_vis.ts

```diff
@@ -122,6 +122,9 @@ class DataVisModel {
     private reportId = ko.observable<string>("Report id: " + reportInfo.rep_id);
     private dataId = ko.observable<string>("Data id: " + reportInfo.dep_id);
     private appId = ko.observable<string>("App. id: " + reportInfo.git_id);
+    private dataLink = ko.observable<string>("https://montagu.vaccineimpact.org/reports/report/"
```

Comment:
This is always going to link to the report on production, so won't work for testing on science (where report versions might exist that don't exist on prod). Maybe that's not an issue if not needed on science? But I guess the root url could be configurable to the environment. Or use a relative url?

Reply:
I don't think it is relevant, the public facing app (and the dependent report) will only ever be on production.